sql-cloudant/README.md

```diff
@@ -31,15 +31,14 @@ The `--packages` argument can also be used with `bin/spark-submit`.
 
 Submit a job in Python:
     
-    spark-submit  --master local[4] --jars <path to cloudant-spark.jar>  <path to python script> 
+    spark-submit  --master local[4] --packages org.apache.bahir:spark-sql-cloudant_2.11:2.2.0-SNAPSHOT  <path to python script> 
```

Comment:
Fixed in 5588c87.  There are existing examples on how to run this command under sections `Using SQL In Python` and `Using SQL In Scala`.